#!/usr/bin/env python
PACKAGE = "bluerov2_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("scale_x",    double_t,    0, "Marker Scale x", 1.0,  0, 400)
gen.add("scale_y",    double_t,    0, "Marker Scale y", 1.0,  0, 400)
gen.add("scale_z",    double_t,    0, "Marker Scale z", 1.0,  0, 400)

gen.add("color_r",    double_t,    0, "Marker Color r", 1.0,  0, 1)
gen.add("color_g",    double_t,    0, "Marker Color g", 0.0,  0, 1)
gen.add("color_b",    double_t,    0, "Marker Color b", 0.0,  0, 1)
gen.add("color_a",    double_t,    0, "Marker Opacity", 1.0,  0, 1)

exit(gen.generate(PACKAGE, "trajectory_visualizer.py", "marker_reconfig"))